enterprise/kernel/src/main/java/org/neo4j/kernel/impl/store/format/highlimit/v300/HighLimitV3_0_0.java

```diff
@@ -68,6 +68,12 @@ public HighLimitV3_0_0()
     }
 
     @Override
+    public String neo4jVersion()
+    {
+        return "3.0.0";
```

Comment:
what instead of store this in each format we will add a field in StoreVersion (introducedIn field) for example and then we will just return that field value from each of the format?